openpgp/packet/public_key.go

```diff
@@ -735,13 +735,13 @@ func (pk *PublicKey) VerifyRevocationSignature(sig *Signature) (err error) {
 }
 
 // VerifySubkeyRevocationSignature returns nil iff sig is a valid subkey revocation signature,
-// made by the passed in signingKey.
-func (pk *PublicKey) VerifySubkeyRevocationSignature(sig *Signature, signingKey *PublicKey) (err error) {
-	h, err := keyRevocationHash(pk, sig.Hash)
+// made by this public key, of signed.
+func (pk *PublicKey) VerifySubkeyRevocationSignature(sig *Signature, signed *PublicKey) (err error) {
+	h, err := keySignatureHash(pk, signed, sig.Hash)
```

Comment:
Is this interface change needed?

Reply:
It is not, however, this function was also broken (it would fail to verify subkey revocation signatures), and unused. So this change is not breaking it any more than it already was, and makes it consistent with the other functions (and actually working :sweat_smile:)

Reply:
It is still exported tho, but fine for me :+1: